insights/parsers/messages.py

```diff
@@ -86,3 +86,23 @@ def get(self, s):
                     msg_info['procname'] = info_splits[4]
                 r.append(msg_info)
         return r
+
+    def get_after(self, timestamp, lines=None):
+        """
+        Get a list of lines after the given time stamp.
+
+        Parameters:
+            timestamp(datetime.datetime): log lines after this time are
+                returned.
+            lines(list): the list of log lines to search (e.g. from a get).
+                If not supplied, all available lines are searched.
+
+        Returns:
+            (list): The list of log lines with time stamps after the given
+                date and time.
+
+        *NOTE*: Because /var/log/messages timestamps by default have no year,
+        you must supply the year in your timestamp as 1900.  This may also
+        cause confusion around the end of the year.
```

Comment:
Do you mean 'test that `get_after()` handles timestamps around 1900 correctly'?  Or '`get_after()` should test that the tested-for timestamp has a year reasonably close to the log's timestamp'?  I'm not sure how to do the former, and I don't think we should do the latter, so obviously I don't understand what you mean here :-)